src/main.cpp

```diff
@@ -49,13 +49,19 @@ int main(int argc, const char** argv)
         // Check for success
         if (e.code())
         {
-            lo2s::Log::error() << "Aborting: " << e.what();
+            lo2s::Log::fatal() << "Aborting: " << e.what();
+            return EXIT_FAILURE;
```

Comment:
I think `return e.code().value()` is actually the sensible thing to do here @tilsche ?

Reply:
sure

Reply:
Please change the return code and we're good to go